include/boost/url/params.hpp

```diff
@@ -217,43 +187,21 @@ class params
     assign(FwdIt first, FwdIt last);
 
 private:
-    /** Assignment from forward iterators
-
-        Assign @ref params from a range of @ref params::value_type.
-
-        Each instance of @ref params::value_type is a view of a query <key, value> pair.
-
-        @tparam FwdIt Forward Iterator Type
-
-        @param first An iterator to the beginning of the query parameter range
-
-        @param last An iterator to the end of the query parameter range
-
-     */
     template<class FwdIt>
     void
     assign(FwdIt first, FwdIt last,
         std::forward_iterator_tag);
 
 public:
 #ifndef BOOST_URL_DOCS
-    /** Assignment from input iterators is deleted
-
-        @note <!-- Doxygen cannot render ` = delete` -->
-     */
+    // Doxygen cannot render ` = delete`
     template<class FwdIt>
     void
     assign(FwdIt first, FwdIt last,
         std::input_iterator_tag) = delete;
```

Comment:
\sigh.. please open an issue in Docca asking for this to be fixed? Is it a Doxygen issue? The more info the better.

Reply:
Anyway, I generated the documentation without the guards, and this is what we get:

![image](https://user-images.githubusercontent.com/5369819/159541506-6d140f7d-8d8a-4e9d-8a72-63dcaf9fb7a1.png)

![image](https://user-images.githubusercontent.com/5369819/159541575-385f8260-c217-4f09-ad30-2b3fa55ff8d0.png)

So I assume this is a docca issue. I opened the issue [here](https://github.com/boostorg/docca/issues/122).